tests/integration.rs

```diff
@@ -362,7 +359,6 @@ fn test_version_sort_overwrite_by_sizesort() {
         .stdout(predicate::str::is_match("11\n2\n$").unwrap());
 }
 
-#[cfg(test)]
```

Comment:
Any reason why you removed this?

Reply:
According to the Rust book, anything in the `tests` directory is only compiled for tests anyway so effectively the whole file is `#[cfg(test)]` and makes any use of it in the `tests` directory redundant.